meta/main.yml

```diff
@@ -0,0 +1,25 @@
+# meta file for nagios-plugin-percona
+---
+galaxy_info:
+  author: Mischa ter Smitten
+  company: Oefenweb.nl B.V.
+  description: Set up Percona Monitoring Plugins for Nagios in Debian-like systems
+  license: MIT
+  min_ansible_version: 1.4
+  platforms:
+    - name: Ubuntu
+      versions:
+        - lucid
+        - precise
+        - trusty
+    - name: Debian
+      versions:
+        - squeeze
+        - wheezy
+        - jessie
+  categories:
```

Comment:
Let's drop support for `lucid` and `squeeze` because as far as I know there're no 10.04 packages anymore.

Can you also remove it in the `Vagrantfile`?